lib/videoroom_web/templates/room/index.html.heex

```diff
@@ -22,8 +22,8 @@
                     <audio></audio>
                     <video class="w-full h-full"></video>
                     <img class="invisible invert" src={static_path(@conn, "/svg/mic-off-fill.svg")} style="position: absolute; left: 0; top: 0; width: 10%, height: 10%">
+                    <img name="camera" class="invisible invert" src={static_path(@conn, "/svg/camera-off-line.svg")} style="position: absolute; top: 0; width: 10%, height: 10%">
```

Comment:
It seems off to combine inline styling with a class (`cam-margin`). I'd use the tailwind's classes or move it to the style-sheet

Reply:
Inline styles were here before my changes/implementation but you're right, I changed them to Tailwind classes.